include/implGBL/EUTelExcludedPlanes.h

```diff
@@ -0,0 +1,28 @@
+#ifndef EUTELEXCLUDEPLANES_H
```

Comment:
Quite right! This was removed in a later commit in another PR. This has been removed now in this PR.